src/lib/charts/index.ts

```diff
@@ -1,2 +1,18 @@
+import ChartDataLabels from 'chartjs-plugin-datalabels';
+import { ChartOptions } from 'chart.js';
+import { merge } from 'lodash';
+import { defaults } from 'react-chartjs-2';
+import './charts.css';
 export * from './PieChart/PieChart';
 export * from './BarChart/BarChart';
+
+// Note: this overrides the default global chart.js options
+const chartDefaultOverrides: ChartOptions = {
+  responsive: true,
+  maintainAspectRatio: true,
+  legend: { position: 'bottom' },
+  plugins: [ChartDataLabels]
+};
+merge(defaults, {
+  global: chartDefaultOverrides
+});
```

Comment:
If we were to create a `ChartBase` component, we could do this within a more React way by doing this through that component, no? It just feels like using a base component would clean up a lot of these odd-ball solutions by allowing us to encapsulate them in a single component/file. That's where we would import our CSS, too, I think.

I'm going to go ahead and get started on that.

If I do that, is there any reason why we couldn't keep the portal, rather than using the helper function? what is the advantage of using `ReactDOM.render` over the portal?

Reply:
The portal was not performant, there were too many rerenders for things when using it with the custom function. This was way more performant for some reason.

Reply:
Also, this is how https://www.npmjs.com/package/react-chartjs-2 recommends updating globals. Why are you calling these global settings an odd-ball solution?

Reply:
It just seems weird having it hanging out in a random file and called outside the component tree.